addons/l10n_ch/static/src/scss/report_swissqr.scss

```diff
@@ -12,6 +12,10 @@ $l10n-ch-qr-ratio: 1.25;
 
 body.l10n_ch_qr, body[data-oe-report="l10n_ch.l10n_ch_qr_report"] {
     padding: 0;
+    span{
+        padding-left:10mm;
+        white-space:nowrap;
+    }
```

Comment:
```suggestion
    span{
        padding-left: 10mm;
        white-space: nowrap;
    }
```

Reply:
I've made this very important change my absolute priority :superhero: